initfiles/sbin/hpcc-run.sh.in

```diff
@@ -52,134 +52,131 @@ getIPS(){
 }
 
 getDali(){
-        DIP=`${INSTALL_DIR}/sbin/configgen -env ${envfile} -listall | grep Dali | awk -F, '{print \$3}'  | sort | uniq`
+    DIP=`${INSTALL_DIR}/sbin/configgen -env ${envfile} -listall | grep Dali | awk -F, '{print \$3}'  | sort | uniq`
 }
 
 createIPListFile(){
-    _file=$1
-    echo "$IPS" > $_file
+    local _output=$1
+    echo "$IPS" > $_output
 }
 
 createIPListFileExcludeDIP(){
-  _file=$1
-  echo "$IPS" | grep -v $DIP  > $_file
+    local _input=$1
+    local _output=$2
+    grep -vwF "$DIP" $_input > $_output 2> /dev/null
```

Comment:
So this is the new code for filtering out the dali IP's.  I tested it on my 4 node cluster with and without more than one dali, and Mark just did the same on his 12 node cluster.  Works as expected.